its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/TelemetryTest.java

```diff
@@ -106,7 +106,11 @@ private void AssertTelemetry(String projectName) throws IOException {
         x -> assertThat(x).isEqualTo("{\"dotnetenterprise.s4net.params.sonar_scanner_scanall.source\":\"CLI\"}"),
         x -> assertThat(x).isEqualTo("{\"dotnetenterprise.s4net.params.sonar_scm_disabled.source\":\"CLI\"}"),
         x -> assertThat(x).isEqualTo("{\"dotnetenterprise.s4net.serverInfo.product\":\"SQ_Server\"}"),
-        x -> assertThat(x).isEqualTo("{\"dotnetenterprise.s4net.serverInfo.serverUrl\":\"custom_url\"}")
+        x -> assertThat(x).isEqualTo("{\"dotnetenterprise.s4net.serverInfo.serverUrl\":\"custom_url\"}"),
+        x -> assertThat(x).isEqualTo("{\"scanner.legacyTFS\":\"NotCalled\"}"),
+        x -> assertThat(x).startsWith("{\"dotnetenterprise.s4net.serverInfo.version\":"),
+        x -> assertThat(x).startsWith("{\"ScannerEngine.NewBootstrapping\":"),
+        x -> assertThat(x).startsWith("{\"ScannerEngine.Download\":")
```

Comment:
Naming:
The `dotnetenterprise.s4net` prefix is missing.

* ScannerEngine.NewBootstrapping -> scannerEngine.newBootstrapping
* ScannerEngine.Download -> scannerEngine.download